src/lib/logger.test.ts

```
@@ -0,0 +1,552 @@
+    it('should detect browser environment correctly', () => {
+      // Simulate browser environment
+      global.window = {} as any
+      
+      // Re-import to get fresh environment check
+      // Note: This is a limitation of the current implementation
+      // In a real scenario, you'd want to make environment detection testable
+      expect(typeof window !== 'undefined').toBe(true)
+    })
+
+    it('should detect Node.js environment correctly', () => {
+      delete global.window
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)